src/main/webapp/app/shared/source/source-dialog.component.html

```diff
@@ -13,6 +13,18 @@ <h4 class="modal-title" id="myDeviceLabel" jhiTranslate="managementPortalApp.sou
                    [(ngModel)]="source.id" readonly />
         </div>
         <div class="form-group">
+            <label class="form-control-label" jhiTranslate="managementPortalApp.source.sourceName" for="field_sourceName">Source Name</label>
+            <input type="text" class="form-control" name="sourceName" id="field_sourceName"
+                   [(ngModel)]="source.sourceName"
+                   required />
+            <div [hidden]="!(editForm.controls.sourceName?.dirty && editForm.controls.sourceName?.invalid)">
+                <small class="form-text text-danger"
+                       [hidden]="!editForm.controls.sourceName?.errors?.required" jhiTranslate="entity.validation.required">
+                    This field is required.
+                </small>
+            </div>
+        </div>
+        <div class="form-group">
             <label class="form-control-label" jhiTranslate="managementPortalApp.source.sourceId" for="field_devicePhysicalId">Source Id</label>
```

Comment:
i think a user cannot enter a `source-id` during creation/edit according to our latest model. So this field should be removed. 
```
<div class="form-group">
            <label class="form-control-label" jhiTranslate="managementPortalApp.source.sourceId" for="field_devicePhysicalId">Source Id</label>
            <input type="text" class="form-control" name="sourceId" id="field_devicePhysicalId"
                [(ngModel)]="source.sourceId"
            required />
            <div [hidden]="!(editForm.controls.sourceId?.dirty && editForm.controls.sourceId?.invalid)">
                <small class="form-text text-danger"
                   [hidden]="!editForm.controls.sourceId?.errors?.required" jhiTranslate="entity.validation.required">
                   This field is required.
                </small>
            </div>
        </div>
```